### ISSUE TITLE
Not installed message in xscreensaver-demo

### ISSUE THREAD

**Issue by alfonsonv** [graysky2/xscreensaver-aerial#34]
I'm using Ubuntu 22.04, follow all the steps, when execute xscreensaver-demo and select ATV4-2K a message appears that says that it is not installed.
https://imgur.com/oBVfFkA

**Comment by graysky2** [comment]
Post:
```
stat /usr/lib/xscreensaver/atv4-2k
```

**Comment by alfonsonv** [comment]
```
~  stat /usr/lib/xscreensaver/atv4-2k
  Fichero: /usr/lib/xscreensaver/atv4-2k
  Tamaño: 4096      	Bloques: 8          Bloque E/S: 4096   directorio
Dispositivo: 803h/2051d	Nodo-i: 5003289     Enlaces: 2
Acceso: (0755/drwxr-xr-x)  Uid: (    0/    root)   Gid: (    0/    root)
Acceso: 2022-05-21 14:05:47.520844703 -0700
Modificación: 2022-05-02 13:04:05.076457156 -0700
      Cambio: 2022-05-02 13:04:05.076457156 -0700
    Creación: 2022-05-02 12:54:07.165910625 -0700
```

**Comment by graysky2** [comment]
The permissions on the file look right... post your `~/.xscreensaver`

**Comment by alfonsonv** [comment]
```
 alfonso@asus  ~  ~/.xscreensaver      
/home/alfonso/.xscreensaver: 5: timeout:: not found
/home/alfonso/.xscreensaver: 6: cycle:: not found
/home/alfonso/.xscreensaver: 7: lock:: not found
/home/alfonso/.xscreensaver: 8: lockTimeout:: not found
/home/alfonso/.xscreensaver: 9: passwdTimeout:: not found
/home/alfonso/.xscreensaver: 10: visualID:: not found
/home/alfonso/.xscreensaver: 11: installColormap:: not found
/home/alfonso/.xscreensaver: 12: verbose:: not found
/home/alfonso/.xscreensaver: 13: timestamp:: not found
/home/alfonso/.xscreensaver: 14: splash:: not found
/home/alfonso/.xscreensaver: 15: splashDuration:: not found
/home/alfonso/.xscreensaver: 16: demoCommand:: not found
/home/alfonso/.xscreensaver: 17: prefsCommand:: not found
/home/alfonso/.xscreensaver: 18: nice:: not found
/home/alfonso/.xscreensaver: 19: memoryLimit:: not found
/home/alfonso/.xscreensaver: 20: fade:: not found
/home/alfonso/.xscreensaver: 21: unfade:: not found
/home/alfonso/.xscreensaver: 22: fadeSeconds:: not found
/home/alfonso/.xscreensaver: 23: fadeTicks:: not found
/home/alfonso/.xscreensaver: 24: captureStderr:: not found
/home/alfonso/.xscreensaver: 25: ignoreUninstalledPrograms:False: not found
/home/alfonso/.xscreensaver: 26: font:: not found
/home/alfonso/.xscreensaver: 27: dpmsEnabled:: not found
/home/alfonso/.xscreensaver: 28: dpmsQuickOff:: not found
/home/alfonso/.xscreensaver: 29: dpmsStandby:: not found
/home/alfonso/.xscreensaver: 30: dpmsSuspend:: not found
/home/alfonso/.xscreensaver: 31: dpmsOff:: not found
/home/alfonso/.xscreensaver: 32: grabDesktopImages:: not found
/home/alfonso/.xscreensaver: 33: grabVideoFrames:: not found
/home/alfonso/.xscreensaver: 34: chooseRandomImages:: not found
/home/alfonso/.xscreensaver: 35: imageDirectory:: not found
/home/alfonso/.xscreensaver: 37: mode:: not found
/home/alfonso/.xscreensaver: 38: selected:: not found
/home/alfonso/.xscreensaver: 40: textMode:: not found
/home/alfonso/.xscreensaver: 41: textLiteral:: not found
/home/alfonso/.xscreensaver: 42: textFile:: not found
/home/alfonso/.xscreensaver: 43: textProgram:: not found
/home/alfonso/.xscreensaver: 44: textURL:: not found
/home/alfonso/.xscreensaver: 46: programs:: not found
/home/alfonso/.xscreensaver: 292: pointerPollTime:: not found
/home/alfonso/.xscreensaver: 293: pointerHysteresis:: not found
/home/alfonso/.xscreensaver: 294: windowCreationTimeout:0:00:30: not found
/home/alfonso/.xscreensaver: 295: initialDelay:: not found
/home/alfonso/.xscreensaver: 296: GetViewPortIsFullOfLies:False: not found
/home/alfonso/.xscreensaver: 297: procInterrupts:: not found
/home/alfonso/.xscreensaver: 298: xinputExtensionDev:: not found
/home/alfonso/.xscreensaver: 299: overlayStderr:: not found
/home/alfonso/.xscreensaver: 300: authWarningSlack:: not found

```

**Comment by graysky2** [comment]
I think you tried to run that file.  Copy the contents of it using a tool like `xclip` or the like

**Comment by alfonsonv** [comment]
Yes, I tried to run it, sorry. 
Te content of the file is a bit long, y put it in pastebin:
https://pastebin.com/NhYSn6BG

**Comment by graysky2** [comment]
After reviewing your config file, I'm sure sure why you're getting that..

**Comment by alfonsonv** [comment]
Can you tell me why?

**Comment by daehenoc** [comment]
Hi, I think this issue may be caused by Ubuntu 22.04 using /usr/libexec/xscreensaver for storing these files, instead of using /usr/lib/xscreensaver.

I recently moved to 22.04 myself and noticed that xscreensaver-aerial was giving a strange error, after digging around today I found the path had changed.  I moved the atv4-2k file from /usr/lib/xscreensaver (which is now empty apart from the atv4-2k file) into /usr/libexec/xscreensaver (which has all the other xscreensaver files in it now) and everything worked OK.
